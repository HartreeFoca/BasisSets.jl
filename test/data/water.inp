!HF STO-3G PRINTBASIS PRINTMOS
*xyz 0 1
O 0.000000 -0.007156 0.965491
H 0.000000 0.001486 -0.003471
H 0.000000 0.931026 1.207929
*
