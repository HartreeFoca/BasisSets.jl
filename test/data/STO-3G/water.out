
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ######   ,####,        
            ##       ##  ##   ,#'  ##            #'  '#       #        #'  '#        
            ##       ##  #######   ##           ,######,      #####,   #    #        
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,         ##   #,  ,#        
             '#######'   ##     ##  '#######'  #'      '#     #####' # '####'        



                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #    Directorship and core code : Frank Neese         #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 5.0.3 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM and meta-GGA Hessian, CC/C-PCM, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : MC-RPA, TRAH-SCF, COSX integrals
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Marcus Kettner         : VPT2
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : Initial AUTO-CI
   Lucas Lang             : DCDCAS
   Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR, DLPNO-MP2 response
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse, P. Pracht,  : VdW corrections, initial TS optimization,
                  C. Bannwarth, S. Ehlert          DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, DLPNO-Multilevel, CI-OPT
                                                   MM, QMMM, 2- and 3-layer-ONIOM, Crystal-QMMM,
                                                   LR-CPCM, SF, NACMEs, symmetry and pop. for TD-DFT,
                                                   nearIR, NL-DFT gradient (VV10), updates on ESD,
                                                   ML-optimized integration grids
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library
   Liviu Ungur et al                             : ANISO software


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 5.1.0
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices


================================================================================

----- Orbital basis set information -----
Your calculation utilizes the basis: STO-3G
   H-Ne       : W. J. Hehre, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2657 (1969).
   Na-Ar      : W. J. Hehre, R. Ditchfield, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2769 (1970).
   K,Ca,Ga-Kr : W. J. Pietro, B. A. Levy, W. J. Hehre and R. F. Stewart, J. Am. Chem. Soc. 19, 2225 (1980).
   Sc-Zn,Y-Cd : W. J. Pietro and W. J. Hehre, J. Comp. Chem. 4, 241 (1983).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================

WARNING: Old DensityContainer found on disk!
         Will remove this file - 
         If you want to keep old densities, please start your calculation with a different basename. 


INFO   : the flag for use of the SHARK integral package has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = water.inp
|  1> !HF STO-3G PRINTBASIS PRINTMOS
|  2> *xyz 0 1
|  3> O 0.000000 -0.007156 0.965491
|  4> H 0.000000 0.001486 -0.003471
|  5> H 0.000000 0.931026 1.207929
|  6> *
|  7> 
|  8>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  O      0.000000   -0.007156    0.965491
  H      0.000000    0.001486   -0.003471
  H      0.000000    0.931026    1.207929

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 O     8.0000    0    15.999    0.000000   -0.013523    1.824514
   1 H     1.0000    0     1.008    0.000000    0.002808   -0.006559
   2 H     1.0000    0     1.008    0.000000    1.759384    2.282655

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     0.969000537465     0.00000000     0.00000000
 H      1   2   0     0.969000334865   103.97799080     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     1.831145639432     0.00000000     0.00000000
 H      1   2   0     1.831145256572   103.97799080     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 6s3p contracted to 2s1p pattern {33/3}
 Group   2 Type H   : 3s contracted to 1s pattern {3}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2

-------------------------
BASIS SET IN INPUT FORMAT
-------------------------

 # Basis set for element : H 
 NewGTO H 
 S 3 
   1       3.4252509100      0.1543289707
   2       0.6239137300      0.5353281424
   3       0.1688554000      0.4446345420
  end;

 # Basis set for element : O 
 NewGTO O 
 S 3 
   1     130.7093200000      0.1543289687
   2      23.8088610000      0.5353281356
   3       6.4436083000      0.4446345363
 S 3 
   1       5.0331513000     -0.0999672287
   2       1.1695961000      0.3995128246
   3       0.3803890000      0.7001154606
 P 3 
   1       5.0331513000      0.1559162685
   2       1.1695961000      0.6076837141
   3       0.3803890000      0.3919573862
  end;


Checking for AutoStart:
The File: water.gbw exists
Trying to determine its content:
     ... Fine, the file contains calculation information
     ... Fine, the calculation information was read
     ... Fine, the file contains a basis set
     ... Fine, the basis set was read
     ... Fine, the file contains a geometry
     ... Fine, the geometry was read
     ... Fine, the file contains a set of orbitals
     ... Fine, the orbitals can be read
     => possible old guess file was deleted
     => GBW file was renamed to GES file
     => GES file is set as startup file
     => Guess is set to MORead
     ... now leaving AutoStart

------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------
------------------------------------------------------------------------------
                   ___                                                        
                  /   \      - P O W E R E D   B Y -                         
                 /     \                                                     
                 |  |  |   _    _      __       _____    __    __             
                 |  |  |  | |  | |    /  \     |  _  \  |  |  /  |          
                  \  \/   | |  | |   /    \    | | | |  |  | /  /          
                 / \  \   | |__| |  /  /\  \   | |_| |  |  |/  /          
                |  |  |   |  __  | /  /__\  \  |    /   |      \           
                |  |  |   | |  | | |   __   |  |    \   |  |\   \          
                \     /   | |  | | |  |  |  |  | |\  \  |  | \   \       
                 \___/    |_|  |_| |__|  |__|  |_| \__\ |__|  \__/        
                                                                              
                      - O R C A' S   B I G   F R I E N D -                    
                                      &                                       
                       - I N T E G R A L  F E E D E R -                       
                                                                              
 v1 FN, 2020, v2 2021                                                         
------------------------------------------------------------------------------


Reading SHARK input file water.SHARKINP.tmp ... ok
----------------------
SHARK INTEGRAL PACKAGE
----------------------

Number of atoms                             ...      3
Number of basis functions                   ...      7
Number of shells                            ...      5
Maximum angular momentum                    ...      1
Integral batch strategy                     ... SHARK/LIBINT Hybrid
RI-J (if used) integral strategy            ... SPLIT-RIJ (Revised 2003 algorithm where possible)
Printlevel                                  ...      1
Contraction scheme used                     ... SEGMENTED contraction
Coulomb Range Separation                    ... NOT USED
Exchange Range Separation                   ... NOT USED
Finite Nucleus Model                        ... NOT USED
Auxiliary Coulomb fitting basis             ... NOT available
Auxiliary J/K fitting basis                 ... NOT available
Auxiliary Correlation fitting basis         ... NOT available
Auxiliary 'external' fitting basis          ... NOT available
Integral threshold                          ...     1.000000e-10
Primitive cut-off                           ...     1.000000e-11
Primitive pair pre-selection threshold      ...     1.000000e-11

Calculating pre-screening integrals         ... done (  0.0 sec) Dimension = 5
Organizing shell pair data                  ... done (  0.0 sec)
Shell pair information
Total number of shell pairs                 ...        15
Shell pairs after pre-screening             ...        15
Total number of primitive shell pairs       ...       135
Primitive shell pairs kept                  ...       135
          la=0 lb=0:     10 shell pairs
          la=1 lb=0:      4 shell pairs
          la=1 lb=1:      1 shell pairs

Calculating one electron integrals          ... done (  0.0 sec)
Calculating Nuclear repulsion               ... done (  0.0 sec) ENN=      9.084261638166 Eh

SHARK setup successfully completed in   0.0 seconds

Maximum memory used throughout the entire GTOINT-calculation: 3.9 MB
-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... water
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   10
 Basis Dimension        Dim             ....    7
 Nuclear Repulsion      ENuc            ....      9.0842616382 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Trust-Rad. Augm. Hess. CNVTRAH         .... auto
   Auto Start mean grad. ratio tolernc. ....  1.125000
   Auto Start start iteration           ....     1
   Auto Start num. interpolation iter.  ....    10
   Max. Number of Micro iterations      ....    16
   Max. Number of Macro iterations      .... Maxiter - #DIIS iter
   Number of Davidson start vectors     ....     2
   Converg. threshold I  (grad. norm)   ....   5.000e-05
   Converg. threshold II (energy diff.) ....   1.000e-06
   Grad. Scal. Fac. for Micro threshold ....   0.100
   Minimum threshold for Micro iter.    ....   0.010
   NR start threshold (gradient norm)   ....   0.001
   Initial trust radius                 ....   0.400
   Minimum AH scaling param. (alpha)    ....   1.000
   Maximum AH scaling param. (alpha)    .... 1000.000
   Orbital update algorithm             .... Taylor
   White noise on init. David. guess    .... on
   Maximum white noise                  ....   0.010
   Quad. conv. algorithm                .... NR
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... SHARK and LIBINT hybrid scheme
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 3.499e-01
Time for diagonalization                   ...    0.000 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.000 sec
Total time needed                          ...    0.000 sec

---------------------
INITIAL GUESS: MOREAD
---------------------
Guess MOs are being read from file: water.ges
Input Geometry matches current geometry (good)
Atom 0: N(Shells)= 3 and 5 - projection required
Input basis set needs projection
NBasis(read)= 13
NBasis(calc)= 7
Basis set overlap was calculated
Using GuessMode=FMatrix for the initial guess
C*T matrix was formed
Effective Fockian was calculated
Effective Fockian was diagonalized
Initial density was built
                      ------------------
                      INITIAL GUESS DONE (   0.0 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
  0    -74.9550872619   0.000000000000 0.02658357  0.00764350  0.0856319 0.7000
  1    -74.9578974748  -0.002810212924 0.02277315  0.00662663  0.0627358 0.7000
                               ***Turning on DIIS***
  2    -74.9598883104  -0.001990835601 0.05443215  0.01612888  0.0433508 0.0000
  3    -74.9535877044   0.006300606032 0.00870922  0.00161287  0.0042629 0.0000
                      *** Initiating the SOSCF procedure ***
                           *** Shutting down DIIS ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  4    -74.96331809  -0.0097303886  0.002004  0.002004  0.005359  0.001051
               *** Restarting incremental Fock matrix formation ***
  5    -74.96444754  -0.0011294479  0.000617  0.000588  0.001456  0.000334
  6    -74.96444885  -0.0000013045  0.000263  0.000443  0.001026  0.000246
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   7 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :          -74.96444915 Eh           -2039.88637 eV

Components:
Nuclear Repulsion  :            9.08426164 Eh             247.19533 eV
Electronic Energy  :          -84.04871078 Eh           -2287.08169 eV
One Electron Energy:         -122.17412627 Eh           -3324.52699 eV
Two Electron Energy:           38.12541548 Eh            1037.44530 eV

Virial components:
Potential Energy   :         -149.52617278 Eh           -4068.81402 eV
Kinetic Energy     :           74.56172363 Eh            2028.92765 eV
Virial Ratio       :            2.00540124


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -2.9991e-07  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    1.7496e-05  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    3.8626e-06  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    1.3107e-05  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    6.8769e-06  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (water.gbw) ****
             **** DENSITY water.scfp WAS UPDATED ****
             **** ENERGY FILE WAS UPDATED (water.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (water.gbw) ****
             **** DENSITY water.scfp WAS UPDATED ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -20.243918      -550.8650 
   1   2.0000      -1.263083       -34.3702 
   2   2.0000      -0.610862       -16.6224 
   3   2.0000      -0.452873       -12.3233 
   4   2.0000      -0.390909       -10.6372 
   5   0.0000       0.594960        16.1897 
   6   0.0000       0.726922        19.7806 
------------------
MOLECULAR ORBITALS
------------------
                      0         1         2         3         4         5   
                 -20.24392  -1.26308  -0.61086  -0.45287  -0.39091   0.59496
                   2.00000   2.00000   2.00000   2.00000   2.00000   0.00000
                  --------  --------  --------  --------  --------  --------
  0O   1s         0.994159  0.233211  0.000000  0.102886  0.000000 -0.130474
  0O   2s         0.026308 -0.837732 -0.000000 -0.534559 -0.000000  0.863105
  0O   1pz       -0.002586  0.076747  0.481419 -0.470249 -0.000000 -0.453244
  0O   1px       -0.000000  0.000000 -0.000000  0.000000 -1.000000 -0.000000
  0O   1py        0.003371 -0.100019  0.369406  0.612841  0.000000  0.590678
  1H   1s        -0.005837 -0.157787 -0.446075  0.283114  0.000000 -0.788068
  2H   1s        -0.005837 -0.157787  0.446075  0.283114  0.000000 -0.788067
                      6   
                   0.72692
                   0.00000
                  --------
  0O   1s        -0.000000
  0O   2s         0.000001
  0O   1pz        0.779332
  0O   1px        0.000000
  0O   1py        0.598003
  1H   1s         0.828439
  2H   1s        -0.828441



                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 O :   -0.354525
   1 H :    0.177263
   2 H :    0.177263
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 O s       :     3.831932  s :     3.831932
      pz      :     1.211416  p :     4.522593
      px      :     2.000000
      py      :     1.311177
  1 H s       :     0.822737  s :     0.822737
  2 H s       :     0.822737  s :     0.822737


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 O :   -0.246363
   1 H :    0.123182
   2 H :    0.123182

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 O s       :     3.682311  s :     3.682311
      pz      :     1.234192  p :     4.564053
      px      :     2.000000
      py      :     1.329861
  1 H s       :     0.876818  s :     0.876818
  2 H s       :     0.876818  s :     0.876818


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 O      8.3545     8.0000    -0.3545     1.9135     1.9135     0.0000
  1 H      0.8227     1.0000     0.1773     0.9686     0.9686    -0.0000
  2 H      0.8227     1.0000     0.1773     0.9686     0.9686    -0.0000

  Mayer bond orders larger than 0.100000
B(  0-O ,  1-H ) :   0.9568 B(  0-O ,  2-H ) :   0.9568 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.071 sec
Sum of individual times     ....       0.006 sec  (  8.4%)

Fock matrix formation       ....       0.004 sec  (  5.8%)
Diagonalization             ....       0.000 sec  (  0.1%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.001 sec  (  1.4%)
Initial guess               ....       0.000 sec  (  0.6%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.3%)
SOSCF solution              ....       0.000 sec  (  0.1%)

Maximum memory used throughout the entire SCF-calculation: 223.5 MB

-------------------------   --------------------
FINAL SINGLE POINT ENERGY       -74.964449145338
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... water.gbw
Electron density                                ... water.scfp
The origin for moment calculation is the CENTER OF MASS  = ( 0.000000,  0.086591  1.747693)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:      0.00000      -0.25321       0.19430
Nuclear contribution   :      0.00000       0.78810      -0.60473
                        -----------------------------------------
Total Dipole Moment    :      0.00000       0.53489      -0.41043
                        -----------------------------------------
Magnitude (a.u.)       :      0.67421
Magnitude (Debye)      :      1.71370



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:    26.442494    14.345791     9.300183 
Rotational constants in MHz : 792726.023576 430076.000197 278812.457653 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :     0.000000    -0.674209     0.000000 
x,y,z [Debye]:     0.000000    -1.713704     0.000000 

 

Timings for individual modules:

Sum of individual times         ...        0.149 sec (=   0.002 min)
GTO integral calculation        ...        0.050 sec (=   0.001 min)  33.5 %
SCF iterations                  ...        0.099 sec (=   0.002 min)  66.5 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 0 seconds 209 msec
